src/components/Editor/Runners/HtmlRunner/HtmlRunner.jsx

```diff
@@ -104,21 +113,47 @@ function HtmlRunner() {
       if (typeof event.data?.msg === "string") {
         if (event.data?.msg === "ERROR: External link") {
           setError("externalLink");
-        } else {
+        } else if (event.data?.msg !== "Allowed external link") {
```

Comment:
Can we switch these around to make it clearer? I.e. can we have `if (event.data?.msg === "Allowed external link") <do allowed external link stuff> else <render html from user's code>`